internal/querycoordv2/server.go

```diff
@@ -516,7 +516,7 @@ func (s *Server) recoverCollectionTargets(ctx context.Context, collection int64)
 	err := s.targetMgr.UpdateCollectionNextTarget(collection)
 	if err != nil {
 		msg := "failed to update next target for collection"
-		log.Error(msg, zap.Error(err))
+		log.Warn(msg, zap.Error(err))
```

Comment:
This has to be a error, recovery failure will lead to panic later

Reply:
ok, get